vermouth/gmx/itp_read.py

```diff
@@ -271,7 +271,8 @@ def _interactions(self, line, lineno=0):
             atom_idxs=atom_idxs,
             )
 
-    def _split_atoms_and_parameters(self, tokens, atom_idxs):
+    @staticmethod
+    def _split_atoms_and_parameters(tokens, atom_idxs):
         """
```

Comment:
I can't exactly remember why but for some inheritance reasons these were not static methods of the class. How badly do you wanna change this? I would advertise to come back to this as soon as we redo the parsers. I know it seems like almost an equivalent change but it had it's reasons.

Reply:
Same for the next three methods

Reply:
I don't think the staticmethod has any influence on the inheritance behaviour, but I'll change it.